release/Earthfile

```
@@ -99,102 +186,55 @@ release-homebrew:
-        -e 's^ldflags = ".*"^ldflags = "'"$(cat /params/ldflags1)"'"^' \
-        -e '/ldflags = ".*"/{n;s^".*"^"'"$(cat /params/ldflags2)"'"^}' \
+        -e 's^\burl ".*"^url "'"$NEW_URL"'"^' \
+        -e 's^\bsha256 ".*"$^sha256 "'$(cat /params/downloadsha256)'"^' \
+        -e 's^\btags = ".*"^tags = "'"$(cat /params/tags)"'"^' \
+        -e 's^\bldflags = ".*"^ldflags = "'"$(cat /params/ldflags)"'"^' \
         ./Formula/earthly.rb
     RUN echo "Diff:" && git diff
     RUN version=${RELEASE_TAG#v} ;\
         echo version=$version ;\
-        git commit -a -m "earthly $version"
-    
-    # Fork, push branch to fork and create upstream PR.
-    RUN --secret GITHUB_TOKEN=+secrets/earthly-technologies/github/griswoldthecat/token test -n "$GITHUB_TOKEN"
-    RUN --push \
-        --secret GITHUB_TOKEN=+secrets/earthly-technologies/github/griswoldthecat/token \
-        -- \
-        hub fork
-    # Force use of https so we can use GITHUB_TOKEN via envcredhelper.sh.
-    RUN --push \
-        -- \
-        git remote set-url "$GIT_USERNAME" "https://github.com/$GIT_USERNAME/homebrew-earthly.git" && \
-        git remote -v
-    RUN --push \
-        --secret GIT_PASSWORD=+secrets/earthly-technologies/github/griswoldthecat/token \
-        -- \
-        git push "$GIT_USERNAME" "release-$RELEASE_TAG"
-    RUN --push \
-        --secret GIT_PASSWORD=+secrets/earthly-technologies/github/griswoldthecat/token \
-        --secret GITHUB_TOKEN=+secrets/earthly-technologies/github/griswoldthecat/token \
-        --secret SLACK_WEBHOOK_URL=+secrets/earthly-technologies/slack/release-webhook \
-        -- \
-        version=${RELEASE_TAG#v} ;\
-        echo version=$version ;\
-        output=$(hub pull-request --no-edit \
-            -h "$GIT_USERNAME":"release-$RELEASE_TAG" \
-            -b "earthly:main" \
-            -m "earthly $version" \
-            -m '-------------' \
-            -m '#### Debug data' \
-            -m 'PR generated by the [Earthly build](https://github.com/earthly/earthly/blob/main/release/Earthfile) (target +release-homebrew)' \
-            -m '* `RELEASE_TAG='"$RELEASE_TAG"'`' \
-            -m '* `GIT_USERNAME='"$GIT_USERNAME"'`' \
-            -m '* `NEW_URL='"$NEW_URL"'`' \
-            -m '* `NEW_SHA256='"$(cat /params/downloadsha256)"'`' \
-            -m '* `TAGS='"$(cat /params/tags)"'`' \
-            -m '* `LDFLAGS='"$(cat /params/ldflags)"'`' \
-            -m '* `LDFLAGS1='"$(cat /params/ldflags1)"'`' \
-            -m '* `LDFLAGS2='"$(cat /params/ldflags2)"'`') && \
-        echo $output && \
-        curl -X POST -H 'Content-type: application/json' --data '{"text":"Successfully released `'$RELEASE_TAG'`: https://github.com/earthly/earthly/releases/tag/'$RELEASE_TAG'"}' $SLACK_WEBHOOK_URL && \
-        curl -X POST -H 'Content-type: application/json' --data '{"text":"Successfully released homebrew PR for `'$RELEASE_TAG'`: '"$output"'"}' $SLACK_WEBHOOK_URL
+        git commit -a -m "earthly $version" || true
```

`|| true` required to allow this code to be re-entrant in the case of the version already being released.

---

This is probably better

```suggestion
        git commit -a --allow-empty -m "earthly $version"
```